doc/api/buffer.md

```diff
@@ -1349,6 +1349,27 @@ console.log(buf1.toString('latin1'));
 A `TypeError` will be thrown if `string` is not a string or another type
 appropriate for `Buffer.from()` variants.
 
+### Static method: `Buffer.fromTypedArray(typedArray[, offset[, length]])`
```

Comment:
Name suggestion:

```suggestion
### Static method: `Buffer.copyTypedArrayMemory(typedArray[, offset[, length]])`
```

Yeah, it's verbose, but the name _should_ reflect that the method copies memory, and doesn't copy contents or share memory.

Reply:
Went with `Buffer.copyFrom(...)`

Reply:
But that’s still a pretty bad name, because the reasonable default thing for something called “copyFrom” to do is to copy _contents_, not memory.

Reply:
I think we have different opinions on what the reasonable default expected behavior is ;-) ... let's go with `copyBytesFrom(...)`